skyrimnet_chatterbox/skyrimnet_chatterbox.py

```
@@ -68,7 +98,7 @@ def load_model():
     MODEL = load_model_if_needed(model_choice, DEVICE, DTYPE, SUPPORTED_LANGUAGE_CODES )
     return MODEL
 
-def generate(model, text,  language_id="en",audio_prompt_path=None, exaggeration=0.5, temperature=0.8, seed_num=0, cfgw=0, min_p=0.05, top_p=1.0, repetition_penalty=1.2, disable_tqdm=False):
+def generate(model, text,  language_id="en",audio_prompt_path=None, exaggeration=0.5, temperature=0.8, seed_num=0, cfgw=0, min_p=0.05, top_p=1.0, repetition_penalty=1.2, disable_tqdm=False):
```

Ruff (0.15.5)

[warning] Unused function argument: `seed_num`

(ARG001)